File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/switch-batch-invoice-under-assess-reason.ts
import { SwitchInvoiceUnderAssessReasonEnum } from "../enums/switch-invoice-under-assess-reason.enum";

export class SwitchBatchInvoiceUnderAssessReason {
    switchBatchInvoiceId: number;
    switchUnderAssessReason: SwitchInvoiceUnderAssessReasonEnum;
    underAssessReason: string;
    comments: string;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/SwitchBatchInvoiceUnderAssessReason.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class SwitchBatchInvoiceUnderAssessReason : AuditDetails
    {
        public int SwitchBatchInvoiceId { get; set; }
        public SwitchUnderAssessReasonEnum SwitchUnderAssessReason { get; set; }
        public string UnderAssessReason { get; set; }
        public string Comments { get; set; }
    }
}
